lib/heroku/command/pg_backups.rb

```diff
@@ -628,7 +630,7 @@ def hpg_app_client(app_name)
   end
 
   def parse_schedule_time(time_str)
-    hour, tz = time_str.match(/([0-2][0-9]):00 (.*)/) && [ $1, $2 ]
+    hour, tz = time_str.match(/([0-2][0-9]):00 ?(.*)/) && [ $1, $2 ]
```

Comment:
Ok, I tried `Shellwords` and it seems to work, so I changed `spec_helper` to do that. I kept the change making the space optional since having that required seems needlessly pedantic.